R/utils.R

```diff
@@ -3,14 +3,21 @@ can_overwrite <- function(path) {
 
   if (!file.exists(path)) {
     TRUE
-  } else if (interactive() && !yesno("Overwrite `", name, "`?")) {
```

Comment:
I think this could be simplified if:

```R
if (!file.exists(path)) {
  TRUE 
} else {
  nope("Overwrite `", name, "`?")
}
```

But maybe I'm trying to be too clever? Either way, the logic here isn't that clear to me.

Reply:
Your proposal works if you're happy to hit a `stop()` in non-interactive session, as opposed to returning `FALSE`. It causes a few test failures, but just because we're failing for a new reason (new message). It doesn't really change outcomes.